web-console/knikubevirt/Create-vm/step-5-review/wizard-review.md

```diff
@@ -0,0 +1,36 @@
+# Create Virtual Machine Wizard - Review Step
+
+The purpose of this doc is to update the current create VM designs to align with OpenShift and Patternfly 4.
+
+Some things to note:
+
+The “Create virtual machine” action has been updated to “Review and create”.
+Once the user has clicked this button they will taken to the Review screen to verify their settings.
+
+![user clicks review and create button](img/review-and-create-action.png)
+
+The action “Create virtual machine” has been moved to the footer with the other actions.
+This encourages the user to check their settings quickly before creating the VM.
+
+![user checks settings before creating vm](img/check-settings.png)
+
+When the user clicks “Create virtual machine” they will then be taken to the list view where they can see the VM being started.
+This replaces the step where the modal is left opened and requires more clicks from the user. Leaving the modal open only allows them to watch the process in the list view, but they could not cancel it or go back.
```

Comment:
we'll need to update this now since it's no longer in a modal. It would be good to get @jelkosz opinion here but I'd like to see the ability to go back and make changes if there is an error when the VM is being created. If we decide that the user is left with this screen until the creation completes then we should provide designs for both success and error here. I believe there are designs started for this approach already that would be useful here.

Reply:
In this scenario too we could see a user clicking away via "Virtual machines" in the main navigation, if so I think they'd go to the list view and we could show the following screens that show the state of the VM as "Starting" or "Creating".

Reply:
1: I believe the user should be allowed to go back in case of error. We do not have it now but I would say it is more like bug than a feature :) 

2: I believe the edits should not be done in the wizard but in the details pages. We already have add/remove nic/disks and will have edit of description and flavor soon. I would find it very confusing to have also some other way how to edit this same things. I would vote for wizard for creation and than the edits in the VM details tabs.

Reply:
@jelkosz ok, so to confirm they can go back and change anything they want while the Wizard is "opened" but as soon as they close it they edit through the details pages. I think that's the best solution for sure.